core/src/com/unciv/logic/GameInfo.kt

```diff
@@ -91,7 +91,7 @@ class GameInfo : IsPartOfGameInfoSerialization, HasGameInfoSerializationVersion
     companion object {
         /** The current compatibility version of [GameInfo]. This number is incremented whenever changes are made to the save file structure that guarantee that
          * previous versions of the game will not be able to load or play a game normally. */
-        const val CURRENT_COMPATIBILITY_NUMBER = 3
+        const val CURRENT_COMPATIBILITY_NUMBER = 4
```

Comment:
are you sure this breaks compatibility & backwards compatibility can't be implemented in a feasible way? 

Back when Azurite implemented this it was only to be used partly for stuff like preventing people from using an older version of the game compared to other players even if the save file still loads but would then break the save after serializing